release_notes/46-release-notes.adoc

```diff
@@ -0,0 +1,350 @@
+. This object is controlled by a feature flag and can be enabled or disabled by using the `ROX_VULN_MGMT_LEGACY_SNOOZE` environment variable.
+. The format for specifying durations in JSON requests to `v1/nodecves/suppress`, `v1/clustercves/suppress` and `v1/imagecves/suppress` are restricted to a proto JSON format. Only a numeric value representing seconds, with optional fractional seconds for nanosecond precision, followed by the `s` suffix is accepted (for example, "0.300s", "-5400s", or "9900s"). This replaces the current format, which allows a string with a signed sequence of decimal numbers, each with an optional fraction and a unit suffix (for example, "300ms", "-1.5h", or "2h45m"). The currently valid time units "ns", "us" (or "µs"), "ms", "m", and "h" are no longer supported.
+. The `/v1/cve/requests` API for managing vulnerability exceptions is removed. Use the new `/v2/vulnerability-exceptions/` API.
+. The *Dashboard* view under *Vulnerability Management* is deprecated. Use the *Workload CVEs*, *Exception Management*, *Platform CVEs*, and *Node CVEs* views as alternatives.
+
+
+[id="deprecated-features_{context}"]
+=== Deprecated features
+
+The following section provides information about additional deprecated features:
+
+* To unify the response data for stream and unary API requests:
+** The `error` field returned for failed unary API requests is deprecated. Instead of the `error` field, use the `message` field to retrieve error information. The `message` field has the same information as the `error` field.
+//is this deprecated in 4.6 or removed?
+** In this release, Red{nbsp}Hat removed the `grpcCode`, `httpCode`, and `httpStatus` fields in returned error response for gRPC stream APIs. Instead, the response includes a new field, `code` which includes the `grpcCode` data.
+//the 4.5 notes say that these above will be removed in the next release. please confirm.
+
+[id="bug-fixes_{context}"]
+== Bug fixes in version 4.6.0
+
+*Release date*: 2 December 2024
+
+* Fixed incorrect timestamp data for the *First discovered* column on the *Workload CVE* single page when viewing affected images.
+
+* Fixed incorrect CVE counts when viewing an image in the *Vulnerability Management* window that contains CVEs with an unknown severity.
+
+* In runtime monitoring, process names and arguments could cause serialization problems when containing invalid UTF-8 characters. This resulted in error messages in the collector logs. Those characters are now filtered and replaced with a `?` when necessary.
+
+* Fixed a bug when using delegated scanning where newer image metadata and layers were pulled incorrectly for an older image referenced by tag when the image registry contents have changed since deployment. Now, the metadata and layers pulled are based on the digest of the image provided by the container runtime when available, instead of just the tag.
```

Comment:
I think that these would be easier to understand if they followed the default bug fix format, similar to this example: https://docs.google.com/presentation/d/1IRuI_CMcMIgQtjfGG2o1PWugMFv_B_bYCQ0rPUuP40c/edit#slide=id.gb820dbee29_1_49

So, something like: 
* Before this release, the timestamp data displayed incorrectly when viewing affected images in the *First discovered* column of the *Workload CVE* single page view. This update resolves the issue.